File: archives/10886.py
N = int(input())
a = []
for _ in range(N):
    a.append(int(input()))

cnt0 = 0
cnt1 = 0
for i in a:
    if i == 0:
        cnt0 += 1
    else:
        cnt1 += 1
if cnt0 > cnt1 : print("Junhee is not cute!")
else: print("Junhee is cute!")

File: archives/cpp/10886.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);
    
    int N;
    cin >> N;

    int one = 0, zero = 0;
    int temp;
    for (int i = 0; i < N; i++) {
        cin >> temp;
        if (temp == 0) { zero++; }
        else { one++; }
    }

    cout << (one < zero ? "Junhee is not cute!" : "Junhee is cute!");

    return 0;
}